packages/cli/src/commands/deviceprofiles/create.ts

```diff
@@ -12,7 +12,7 @@ import { APICommand, inputAndOutputItem, userInputProcessor } from '@smartthings
 
 import { buildTableOutput } from '../deviceprofiles'
 import { DeviceDefinitionRequest } from './view'
-import { chooseCapability } from '../capabilities'
+import {CapabilityId, chooseCapabilityFiltered} from '../capabilities'
```

Comment:
This is a nit and we should have a linting rule for this (I haven't found the right one yet) but we normally have spaces inside the curly braces here.

Reply:
[object-curly-spacing](https://eslint.org/docs/rules/object-curly-spacing) looks right. I also have VSCode set to format changes on save, which helps a lot.

Reply:
Changed